CONTRIBUTING.md

```diff
@@ -12,6 +12,39 @@ Before submitting code or documentation contributions, you should first complete
 
 Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/percona/grafana-dashboards) (CLA). You can do this using your GitHub account and one click.
 
+## Rules and Guidelines
+
+Development Rules and Best Practices while creating a new dashboard -
```

Comment:
We can't link the internal document ( . 
not sure is the overall PMM doc or if we should put this to the Dashboards repo?

I would vote for placing this to some reference page in the overall PMM documentation (pmm-doc repo )
@catalinaadam  ^ WDYT

Reply:
@rnovikovP - yep, I can move this info to a new topic under the Details chapter where we store the Develop Advisors topic. Then this Contributing file would only reference that new topic.